respondd@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=respondd (respondd status for servers)
+After=syslog.target network-online.target
+
+[Service]
+Type=simple
+User=root
+Group=root
+WorkingDirectory=/opt/respondd-%I
```

Comment:
the path doesn't match the one in the `ext-respondd.service.example`, which I would prefer since it matches the project name.